pkg/operator/starter.go

```diff
@@ -104,6 +106,10 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		controllerContext.EventRecorder,
 	)
 
+	eventWatcher := eventwatch.New().
+		WithEventHandler(operatorclient.TargetNamespace, "LateConnections", terminationobserver.ProcessLateConnectionEvents).
```

Comment:
@mfojtik who's producing this event, asking for a friend :wink:

Reply:
https://github.com/openshift/origin/pull/25126